packages/viz/src/lib/Layer.ts

```diff
@@ -19,9 +19,12 @@ export class Layer {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private _deckLayer: any | undefined;
 
-  constructor(source: string | Source, styles = {}) {
+  constructor(source: string | Source, style?: Style | StyleProperties) {
     this._source = buildSource(source);
-    this._styles = new Style(styles);
+
+    if (style !== undefined) {
```

Comment:
If style is undefined we should get 'default styles', so always `buildStyle`right?